src/Auditable.php

```diff
@@ -131,9 +131,16 @@ protected function auditCreatedAttributes(array &$old, array &$new)
     protected function auditUpdatedAttributes(array &$old, array &$new)
```

Comment:
If we're gonna change this, I prefer to have it done for any (sane) level number, instead of being a quick fix for a user's use case.